hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RDBStore.java

```diff
@@ -134,7 +130,7 @@ public RDBStore(File dbFile, ManagedDBOptions dbOptions,
 
       //create checkpoints directory if not exists.
       checkpointsParentDir =
-          Paths.get(dbLocation.getParent(), "db.checkpoints").toString();
+          Paths.get(dbLocation.getParent(), OM_CHECKPOINT_DIR).toString();
```

Comment:
```suggestion
          dbLocation.getParent() + OM_KEY_PREFIX + OM_CHECKPOINT_DIR;
```

I feel it is unnecessary to get create Path and then get the String from it.